docs/podman-create.1.md

```diff
@@ -710,10 +710,13 @@ Network Namespace - current sysctls allowed:
 
 Note: if you use the --network=host option these sysctls will not be allowed.
 
-**--systemd**=*true|false*
+**--systemd**=*true|false|always*
 
 Run container in systemd mode. The default is *true*.
 
+The value *always* enforces the systemd mode is enforced without
+looking at the executable name.
```

Comment:
Can we make this part of the next paragraph? Maybe have `looking at the executable name. Otherwise, if set to true and the command you are running inside the container is systemd or /usr/sbin/init...`